config/crd/bases/objectstorage.giantswarm.io_buckets.yaml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
are we using the contents of the `config` folder?

Reply:
That's generated by controller-manager

Reply:
I know :smile: 

But if don't use it, it's just noise.

Reply:
Well We do need the CRD and the example but that's pretty much it

Reply:
Removing the unused stuff